docs/pages/blog/base-ui-2024-plans.md

```diff
@@ -0,0 +1,72 @@
+---
+title: An exciting year ahead for Base UI
+description: Base UI will see an exciting year in 2024, including a stable release, plenty of new components, and more. Here’s what to expect.
+date: 2024-02-07T00:00:00.000Z
+authors: ['michaldudak', 'danilo-leal']
+tags: ['Base UI', 'Product']
+card: true
+---
+
+It's been a while since we started work on Base UI, a library of unstyled React UI components and hooks.
+It all started years ago, way before similar products were as widely known as today when we imagined a world where Material UI existed without Material Design.
+
+We're super excited because this world is getting closer than ever.
+This year will see a lot of investment in Base UI as we expand the team (we're hiring!) and focus hard on a stable release (tentatively planned for Q3), which will come full of new components, features, and improvements.
+
+Let's walk through some of the things we're cooking up.
+
+## A larger set of components
+
+Base UI today offers a modest set of components and hooks, including some slightly more complex ones such as Autocomplete and Number Input.
+However, we're conscious that many primitive components that would allow a real-world website or application to use Base UI as its foundation are missing.
+Worry no more, as we're focused on shipping the first versions of all of the components listed below for the stable release:
+
+| To-be-added components |                                                                                                |
+| :--------------------- | ---------------------------------------------------------------------------------------------: |
+| Accordion              | [View&nbsp;the&nbsp;GitHub&nbsp;issue&nbsp;→](https://github.com/mui/material-ui/issues/38037) |
+| Alert Dialog           |                                                              [View the GitHub issue&nbsp;→](/) |
+| Checkbox               |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/38036) |
+| Collapsible            |                                                                   View the GitHub issue&nbsp;→ |
+| Drawer                 |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/38181) |
+| Radio Group            |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/38038) |
+| Tooltip                |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/38045) |
+
```

Comment:
A few words to add about the recent initiative to change the customization API:

## Improved customization API

Currently, Base UI can be customized to your heart's content using the `slots` and `slotProps` props. This API, while powerful, has proven to be less than ideal in certain cases. Most notably, it's too lengthy and hard to author and read when used with libraries such as Tailwind CSS. Additionally, since the `slots` and the corresponding `slotProps` are not related in terms of Typescript types, it's possible to introduce bugs or have the compiler complain about valid code.
To address these issues, we're considering adopting another customization API. We're still fleshing out the details and plan to release an RFC on GitHub in the upcoming days.
We are aware that a significant number of projects depend on the current API. We want to assure you that offering a smooth migration experience is very high on our priority list. Once we know all the details about the new API, we'll figure out the best way to migrate existing projects (with either a codemod, wrapper components exposing the old API, or something else).

Reply:
Right on, love it!